src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -129,74 +116,42 @@ function AppointmentsPageV2({
           }}
         />
       )}
-
-      {showCheetahScheduleButton && (
-        <ScheduleNewProjectCheetah
-          showHomePageRefresh={showHomePageRefresh}
-          startNewAppointmentFlow={() => {
-            recordEvent({
-              event: `${GA_PREFIX}-schedule-project-cheetah-button-clicked`,
-            });
-            startNewAppointmentFlow();
-          }}
-        />
-      )}
-
-      {expressCare.enabled && (
-        <>
-          <>
-            {!isCernerOnlyPatient && (
-              <RequestExpressCare
-                {...expressCare}
-                showHomePageRefresh={showHomePageRefresh}
-                startNewExpressCareFlow={() => {
-                  recordEvent({
-                    event: `${GA_PREFIX}-express-care-request-button-clicked`,
-                  });
-                  startNewExpressCareFlow();
-                }}
-              />
-            )}
-            {expressCare.hasRequests && (
-              <h2 className="vads-u-font-size--h3 vads-u-margin-y--3">
-                Your appointments
-              </h2>
-            )}
-
-            <label className="vads-u-display--inline-block vads-u-margin-top--0 vads-u-margin-right--2">
-              Show by type <span className="sr-only" />
-            </label>
-            <Select
-              options={options}
-              onChange={onChange}
-              aria-labelledby="options"
-            />
-            {routes}
-          </>
-        </>
-      )}
-    </PageLayout>
+      {expressCare.useNewFlow &&
```

Comment:
Further cleaned up some EC logic in here that I didn't notice before. We always have to show EC requests somehow, so the base feature flag doesn't make sense in the context of the new homepage.